templates/data/istio-module-data.yaml

```diff
@@ -121,6 +121,29 @@ templates:
                   local:
                     - job_default
                     - job_post
+              - jobConfig:
+                  name: rel-istio-build
```

Comment:
I think we also want to build module to get the template.yaml on release, since this is not centrally handled, yet.